### ISSUE TITLE
Audiences are showing up in the institutional search

### ISSUE THREAD

**Issue by marijane** [NLightenGroup/nlighten-ontology#67]
**Steps to reproduce:**
1. Navigate to https://alaska.dev.eagle-i.net/institution/
2. Observe the list of resources

**Result**:
Audiences are showing up in the list of resources

**Expected Result:**
Audience is annotated with the SearchFlatten property, which I (perhaps mistakenly) understood would suppress it from the search UI.

**Notes:**
Does this need a different annotation, or do we need to refactor this into a taxonomy?

**Comment by skcheng** [comment]
@marijane (sorry I forgot I could just at mention you instead of assigning) It sounds like Audience shouldn't be an instance create and should be a referenced taxonomy. I imagine there is only a finite number of Audience instances that will be referenced.

SearchFlatten means to take the property and it's instance and pull the information up to the parent for searching. Like searching for properties of a human subject. The properties of the human subject resource is "flattened" into the properties of it's parent resource, i.e. ipsc, for searching purposes.

**Comment by marijane** [comment]
@skcheng, 

The reason why I made it an Instance Create is because I didn't have a list of possible audiences with which to create a taxonomy, so I decided to let SWEET editors define them.  I also liked the fact that people could reuse each other's audiences if it was appropriate to do so.  I suppose we could convert it to a taxonomy now that we have some audiences entered by users.

So SearchFlatten only works on properties?  I do kind of feel like the Audience is a property of the Educational Resources.  What would happen if I annotated the "audience" property instead?
